File: EStates/EStates/ClientApp/app/models/apartament.ts
import { User } from "./user.model";

export class Apartament {
    id: number;
    buildingEntranceId: number;
    buildingId: number;
    name: string;
    description: string;
    parentFloorId: number;
    status: string;
    owners: Array<User>;
}

File: ES.Data/Models/Apartament.cs
using System.Collections.Generic;

namespace ES.Data.Models
{
    public class Apartament
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string Status { get; set; }

        public List<ApplicationUser> Owners { get; set; }
    }
}